src/powershell-native/nativemsh/pwrshplugin/entrypoints.cpp

```diff
@@ -307,11 +310,14 @@ DWORD ReportOperationComplete(WSMAN_PLUGIN_REQUEST *requestDetails, DWORD errorC
 
     DWORD result = EXIT_CODE_SUCCESS;
     PWSTR pwszErrorMessage = NULL;
-    GetFormattedErrorMessage(&pwszErrorMessage, errorCode);
+    if (GetFormattedErrorMessage(&pwszErrorMessage, errorCode) == 0)
+    {
+        swprintf_s(szMessage,_countof(szMessage), L"ReportOperationComplete: %d", errorCode);
+    }
 
     result = WSManPluginOperationComplete(requestDetails, 0, errorCode, pwszErrorMessage);
 
-    if (NULL != pwszErrorMessage)
+    if (NULL != pwszErrorMessage && pwszErrorMessage != szMessage)
```

Comment:
This change will conditionally leak memory. If pwszErrorMessage has a value, it was allocated by GetFormattedErrorMessage and must be freed (via delete[]).

Reply:
Fixed

Reply:
This has not been resolved yet.